packages/core/http/core-http-server/src/versioning/types.ts

```diff
@@ -35,6 +35,8 @@ export type VersionedRouteConfig<Method extends RouteMethod> = Omit<
   options?: Omit<RouteConfigOptions<Method>, 'access'>;
   /** See {@link RouteConfigOptions<RouteMethod>['access']} */
   access: Exclude<RouteConfigOptions<Method>['access'], undefined>;
+  /** A human-readable description of this route */
+  description?: string;
```

Comment:
Proposal: make `description` required.

By making the `description` required, we have the option to automatically generate use-facing docs, a requirement for public APIs.

A description for _every_ route also helps dev because all `internal` routes that adopt the versioned router (pretty much most of them) will also have dev-facing info.